test.js

```diff
@@ -454,6 +454,69 @@ test('close event is only emitted once', async function (t) {
   t.pass()
 })
 
+test('local address', async function (t) {
+  t.plan(4)
+
+  const node = new DHT()
+  t.is(node.localAddress(), null)
+
+  await node.ready()
+  t.is(node.localAddress().host, localIP(4))
```

Comment:
instead of copying the impl, just test that it looks like an ip, otherwise what are we testing 🤣

Reply:
Haha true!